react/components/PickupPointsList.js

```diff
@@ -105,7 +106,11 @@ class PickupPointsList extends PureComponent {
                 className={`${
                   styles.pointsItem
                 } pkpmodal-points-item best-pickupPoint-${pickupPoint.id}`}
-                key={`best-pickupPoint-${pickupPoint.id}`}>
+                key={`best-pickupPoint-${pickupPoint.id}`}
+                onClick={() => {
+                  pickupPointSelectionEvent({ selectionMethod: SELECTION_METHOD_LIST })
```

Comment:
the event is added on the wrapper div for two reasons:

- it can differentiate between the "best" pickup points and the "rest"
- the PickupPointInfo component is used elsewhere, where it shouldn't be interactive (it's in fact interactive, but it fires the selection event for the same pickup point that is currently selected, so it's moot, but that's another discussion)